README.md

```diff
@@ -1,6 +1,28 @@
 # MLOps-Projector
 
 
+## Deploy Airflow using Docker Compose
+
+1. Fetch docker-compose.yaml 
+
+```
+wget "https://airflow.apache.org/docs/apache-airflow/2.6.0/docker-compose.yaml"
```

Comment:
we can commit this file to the repo just in case

Reply:
oo, I see you have it already in the repo, fix readme please

Reply:
fixed